README.md

```diff
@@ -73,12 +75,46 @@ for production but for development, it's likely not a huge issue.
 
 > *Tip*: To run this, add `sudo -u postgres ` to the beginning of the following command.
 
-`createuser houdini_user -s -d -P`
+`createuser admin -s -d -P`
```

Comment:
@wwahammy I forget - do we need `houdini_user` AND `admin` roles? Or just `admin` now

Reply:
Just `admin`.